libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -953,11 +953,47 @@ export interface MeetingInfo {
     organizer: TeamsChannelAccount;
 }
 
+// @public
+export type MeetingNotification = TargetedMeetingNotification;
+
+// @public
+export interface MeetingNotificationBase<T> {
+    type: string;
+    value: T;
+}
+
+// @public
+export interface MeetingNotificationChannelData {
+    onBehalfOf?: OnBehalfOf[];
+}
+
+// @public
+export interface MeetingNotificationRecipientFailureInfo {
+    errorCode: string;
+    failureReason: string;
+    recipientMri: string;
+}
+
+// @public
+export interface MeetingNotificationResponse {
+    recipientsFailureInfo?: MeetingNotificationRecipientFailureInfo[];
+}
+
+// @public (undocumented)
+export interface MeetingStageSurface<T> {
+    content: T;
+    contentType: 'task' | string;
```

Comment:
The way it's defined is already like an enum, but not really. If it was just `contentType: 'task'`, then any object of this type would be required to have the value as "task", otherwise the compiler will complain. Currently `contentType` can be any string and the "task" is just a guide. I could change it to `contentType: "task" | "unkown"` and it will behave in the exact way you are describing. The reason I have it this way is because Yunny wanted to allow for flexibility. Is this what we have for the dotnet sdk? If so then I'll make the change right away.

Reply:
Hi @singhk97 , in .net, `contentType `is a enum type with `Unknown `and `Task`

Reply:
Ok makes sense, will update the type in the next iteration

Reply:
Hi @yingduyingdu, In an offline discussion with Yunny I found out that passing `unknown` in the payload would return a 4xx response. And we shouldn't promote something that doesn't work (i.e by making it one of the possible types). She also mentioned that the reason it's there in the dotnet repo is because of a C# enum idiosyncrasy. For this reason will simply keep the type as `contentType: 'task'`.